packages/design-preview/src/components/index.tsx

```diff
@@ -144,6 +146,7 @@ const DesignPreview = ( props: DesignPreviewProps ) => (
 		siteId={ props.siteId }
 		stylesheet={ props.stylesheet }
 		placeholder={ null }
+		isContextOptional={ !! props.isExternallyManaged }
```

Comment:
I find this pattern a bit odd. If we already know we don't want the GS context, can we just not render it?

```
if ( props.isExternallyManaged ) {
  return <Preview { ...props } />;
}

return (
  <GlobalStylesProvider
    siteId={ props.siteId }
    stylesheet={ props.stylesheet }
    placeholder={ null }
  >
    <Preview { ...props } />
  </GlobalStylesProvider>
);
```

Reply:
I agree this is a much better and more straightforward approach.

Applied it here: https://github.com/Automattic/wp-calypso/pull/80925